admin/c2cgeoportal_admin/views/layers_wms.py

```diff
@@ -25,8 +25,16 @@ class LayerWmsViews(AbstractViews):
         ListField('time_mode'),
         ListField('time_widget'),
         ListField('ogc_server', renderer=lambda layer_wms: layer_wms.ogc_server.name),
-        ListField('restrictionareas', renderer=lambda layer_wms:
-                  ', '.join([r.name or '' for r in layer_wms.restrictionareas]))]
+        ListField('interfaces', sortable=False, renderer=lambda layer_wms:
+                  ', '.join([i.name or '' for i in layer_wms.interfaces])),
```

Comment:
Can you put all parameters on one line or all on different lines : 
```
ListField('interfaces', sortable=False, renderer=lambda layer_wms: ', '.join([i.name or '' for i in layer_wms.interfaces])),
```
or for example:
```
ListField('interfaces',
               sortable=False,
               renderer=lambda layer_wms: ', '.join([i.name or '' for i in layer_wms.interfaces])),
```

Reply:
See what sbrunner has done in role